plugins/UsersManager/tests/Integration/APITest.php

```diff
@@ -387,6 +387,49 @@ public function test_updateUser_failsIfPasswordTooLong()
         $this->api->updateUser($this->login, str_pad('foo', UsersManager::PASSWORD_MAX_LENGTH + 1), 'email@example.com', false, $this->password);
     }
 
+    public function test_update_user_fails_if_email_exists_as_other_user_username()
+    {
+        $this->expectException(\Exception::class);
+        $this->expectExceptionMessage('UsersManager_ExceptionEmailExistsAsLogin');
+
+        $user2 = 'existed@example.com';
+        $this->api->addUser($user2, 'password', 'userlogin2@password.de');
+
+        $this->api->updateUser($this->login, $this->password,  'existed@example.com', false, $this->password);
```

Comment:
could reuse the variable so it's easy to see it's the same value
```suggestion
        $this->api->updateUser($this->login, $this->password, $user2, false, $this->password);
```